### ISSUE TITLE
relax cartopy pin?

### ISSUE THREAD

**Issue by mathause** [C2SM/pyvis#131]
For the newest version of mplotutils I bumped the requirements to `cartopy >= 0.22`. However, we set 

https://github.com/C2SM/pyvis/blob/f7e509b9627d36cb09d4eee9ed1f3a84c6eeadda/environment.yml#L17

(not sure why - iconarray?).

@AnnikaLau can this be relaxed? Or do you know what the issue is?

**Comment by AnnikaLau** [comment]
@mathause yes I guess that was because of iconarray. Feel free to change it.
There were some changes in the psyplot library since the last course, so I will have to change the requirements anyway and test everything before the next course.

But maybe we can leave this issue open as a reminder.

**Comment by mathause** [comment]
Thanks for the feedback - fingers crossed that it works with newer cartopy versions & happy to help debug.
